sled-agent/src/illumos/running_zone.rs

```diff
@@ -149,13 +150,13 @@ impl RunningZone {
                 // TODO(https://github.com/oxidecomputer/omicron/issues/725)
                 //
                 // Re-initialize guest_vnic state by inspecting the zone.
-                guest_vnics: vec![],
+                opte_ports: vec![],
             },
         })
     }
 
-    pub fn get_guest_vnics(&self) -> &Vec<Vnic> {
-        &self.inner.guest_vnics
+    pub fn get_opte_ports(&self) -> &Vec<OptePort> {
+        &self.inner.opte_ports
```

Comment:
Checking my understanding - these still are exclusively for use by the guest, right? Like, we don't have any existing plans to use these ports for underlay networks?

Reply:
I believe that eventually, OPTE ports will support control plane zones that need to communicate with the outside world. This is discussed a bit in [OPTE #127](https://github.com/oxidecomputer/opte/issues/127). But internal/underlay traffic will not cross OPTE.

Reply:
Yes, the v1 goal with OPTE is for it to supply the VPC abstraction to guest instances. However, in the future, it could also be used to supply VPC access to "other things" (*waves hands in air*), whatever that might be. Also OPTE could be used to implement other types of networks on top of the physical network, or even just be used to implement a network function like basic transformations, filtering, etc. That is, OPTE could conceivably be used in a lot of places, but it's primary initial use case is to provide the VPC to the guest.

Reply:
@smklein Yes, these are currently exclusively for guests.